### DISCUSSION TITLE
Is there a way to prevent drawing a square over a single point and visa-versa?

### DISCUSSION THREAD

**Discussion by hman1148** [geoman-io/leaflet-geoman#1493] [General]
I am currently working on a project where one of our client requirements is to prevent users from drawing a square, rectangle, or custom shape over a point and visa-versa. IS there built-in functionality with Geoman where I can check these intersections in a feature group or layer? Thanks!

**Comment by Falke-Design** [comment]
No there is no built in function for this. You can listen on the `pm:create` event and then check by your self if the layer contains the point and if yes, remove the layer from the map again.
